wishlist/css/extra--large-media-query.css

```diff
@@ -0,0 +1,148 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+html,
+body {
+  scroll-behavior: smooth;
+  height: 100vh;
+}
+
+img {
+  max-inline-size: 100%;
+  block-size: auto;
+  object-fit: cover;
+  object-fit: top center;
+}
+
+:root {
+  --primary-light-color: #2565c7;
+  --primary-light-bg-color: #19468b;
+
+  --primary-font-color: #fff;
+  --primary-font-dark-color: #696b79;
+  --primary-dark-bg-color: #1a1a1d;
+}
+
+.no-bg {
+  background: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-box {
+  display: flex;
+  flex-flow: row wrap;
+  box-shadow: rgba(0, 0, 0, 0.05) 0px 6px 24px 0px,
+    rgba(0, 0, 0, 0.08) 0px 0px 0px 1px;
+  padding: 1rem 4rem;
+}
+
+#nav-section {
+  display: flex;
+  flex-flow: row wrap;
+  flex-grow: 1;
+}
+
+#nav-section-mobile,
+#mobile-section {
+  display: none;
+}
+
+#logo {
+  text-align: center;
+}
+
+#logo a {
+  text-decoration: none;
+  text-align: center;
+}
+
+#logo div {
+  color: var(--primary-dark-bg-color);
+  margin-top: 5px;
+  font-weight: bolder;
+  font-family: sans-serif;
+}
+
+#nav-search-bar {
+  display: flex;
```

Comment:
I see you have used id selector as well with class selector, It could cause specificity issues, so try to use one type of selector as much as possible.

Reply:
Ok will make changes in future release